var/spack/repos/builtin/packages/embree/package.py

```diff
@@ -12,7 +12,7 @@ class Embree(CMakePackage):
     url = "https://github.com/embree/embree/archive/v3.5.1.tar.gz"
     generator = 'Ninja'
 
-    version('3.5.2', sha256='245af8820a820af94679fa1d43a05a9c825451be0d603b6165229556adf49517')
+    version('3.5.2-1', sha256='245af8820a820af94679fa1d43a05a9c825451be0d603b6165229556adf49517')
```

Comment:
Ok, in this case it doesnt help because it will try to fetch the tarball with `-1` in name.

```suggestion
    version('3.5.2-1', sha256='245af8820a820af94679fa1d43a05a9c825451be0d603b6165229556adf49517',
                  url="https://github.com/embree/embree/archive/v3.5.2.tar.gz")
```

bit verbose though...